frontend-maven-plugin/src/it/custom-install-directory/verify.groovy

```diff
@@ -1,2 +1,2 @@
-assert new File(basedir, 'target/node').exists() : "Node was not installed in the custom install directory";
+assert new File(basedir, 'target/.buildenv/node').exists() : "Node was not installed in the custom install directory";
```

Comment:
Please add a separate PR for this change to discuss. This has nothing to do with yarn and would break a lot of existing projects.